locales/en/plugin__kuadrant-console-plugin.json

```diff
@@ -85,6 +85,7 @@
   "Reference to an existing secret resource containing DNS provider credentials and configuration": "Reference to an existing secret resource containing DNS provider credentials and configuration",
   "Release Notes": "Release Notes",
   "Save": "Save",
+  "Search across name, namespace and/or resource type": "Search across name, namespace and/or resource type",
```

Comment:
Wonder could we shorten this to fit the input?

maybe:
"Search by name, namespace or type"